clang/lib/Analysis/FlowSensitive/DataflowEnvironment.cpp

```diff
@@ -166,17 +166,20 @@ static Value *joinDistinctValues(QualType Type, Value &Val1,
   return JoinedVal;
 }
 
-// When widening does not change `Current`, return value will equal `&Prev`.
-static Value &widenDistinctValues(QualType Type, Value &Prev,
-                                  const Environment &PrevEnv, Value &Current,
-                                  Environment &CurrentEnv,
-                                  Environment::ValueModel &Model) {
+namespace {
+using WidenResult = Environment::ValueModel::WidenResult;
+}
+
+static WidenResult widenDistinctValues(QualType Type, Value &Prev,
```

Comment:
```suggestion
static Environment::ValueModel::WidenResult widenDistinctValues(QualType Type, Value &Prev,
```

AFAICT, this is used exactly once. Maybe just spell it out?